From: Marcin Juszkiewicz <marcin.juszkiewicz@linaro.org>

Upstream-Status: Inappopriate [upstream has it done in other way]

Problem is solved upstream but we have quite old version so backporting patches
is waste of time.

---
 build/make/configure.sh |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- libvpx-v0.9.5.orig/build/make/configure.sh
+++ libvpx-v0.9.5/build/make/configure.sh
@@ -659,12 +659,12 @@ process_common_toolchain() {
         if enabled iwmmxt || enabled iwmmxt2
             then
                 check_add_asflags -mcpu=${tgt_isa}
             elif enabled armv7
             then
-                check_add_cflags -march=armv7-a -mcpu=cortex-a8 -mfpu=neon -mfloat-abi=softfp  #-ftree-vectorize
-        check_add_asflags -mcpu=cortex-a8 -mfpu=neon -mfloat-abi=softfp  #-march=armv7-a
+                check_add_cflags -march=armv7-a -mcpu=cortex-a8 -mfpu=neon   #-ftree-vectorize
+        check_add_asflags -mcpu=cortex-a8 -mfpu=neon   #-march=armv7-a
             else
                 check_add_cflags -march=${tgt_isa}
                 check_add_asflags -march=${tgt_isa}
             fi
 
